### PR TITLE
Change Unique key on albumthing to add thing_type (#187)

### PR DESCRIPTION
**Pull request by tomamplius**
Change Unique key on albumthing to add thing_type #187

### CODE REVIEW

**Comment by derneuere** [comment]
I think the change in album_thing.py is missing:
https://github.com/LibrePhotos/librephotos/blob/072270b6482859f93189619f2b3dc3ddff556383/api/models/album_thing.py#L17-L18

**Comment by tomamplius** [comment]
Ok, we need to migrate to UniqueConstraint?
https://docs.djangoproject.com/en/3.1/ref/models/options/#unique-together
> unique_together. unique_together may be deprecated in the future.

File: api/migrations/0004_fixAlbumThingContraint.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+api/migrations/0004_fixAlbumThingContraint.py
```
Comment (derneuere) [inline api/migrations/0004_fixAlbumThingContraint.py]: I am not sure why this is at the start of the file. But it leads to an error

File: api/migrations/0004_fixAlbumThingContraint.py [hunk 2]
```diff
@@ -9,6 +11,13 @@ class Migration(migrations.Migration):
...
         migrations.AlterUniqueTogether(
             name='albumthing',
-            unique_together={('title', 'thing_type' , 'owner')},
-        )
+            unique_together=set([]),
+        ),
+        migrations.AddConstraint(
+            model_name="albumthing",
+            constraint=models.UniqueConstraint(
+               fields=['room', 'date'],
```
Comment (derneuere) [inline api/migrations/0004_fixAlbumThingContraint.py]: There is no field called "room" or "date" 😅

File: api/migrations/0004_fixAlbumThingContraint.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+api/migrations/0004_fixAlbumThingContraint.py 
+
```
Comment (derneuere) [inline api/migrations/0004_fixAlbumThingContraint.py]: I forget it always too, but the name scheme in python is always with underscores. Please rename the file to 0004_fix_album_thing_constraint.py

**Review by derneuere — CHANGES_REQUESTED** [review]
Found a couple of things that could be improved :)

**Comment by derneuere** [comment]
Looks good :+1:

**Pull request merged**